src/theme/ThemeReducer.js

```
@@ -1,15 +1,15 @@
 
-const themeInitState = null;
+const initialState = null;
 
-export default function ThemeReducer(state = themeInitState, action) {
+export default function ThemeReducer(state = initialState, action) {
 	switch (action.type) {
-		
-		case CHANGE_THEME: 
+
+		case CHANGE_THEME:
 			const root = document.querySelector(':root');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)